README.md

```diff
@@ -50,6 +50,11 @@ A first step to contributing is to pick from the list of Open Data Hub SIGs. Sta
 
 The [Contributor Guide](./contributing.md) provides help on how to get your ideas and bug fixes seen and accepted.
 
+## Community Release Schedule
+
+- **Release Cadence**: A new release every 3 weeks.
+- **Code Freeze**: The Monday of the release week.
+- **Release Day**: The Tuesday following the code freeze.
```

Comment:
do we want to be so strict on the Tue? or just say day(s) after code freeze

Reply:
hmm yes, that would be nice